fem/nonlinearform.cpp

```diff
@@ -581,6 +581,13 @@ double BlockNonlinearForm::GetEnergyBlocked(const BlockVector &bx) const
          }
       }
 
+   // Free the allocated memory
+   for (int i = 0; i < fes.Size(); ++i)
+   {
+      delete el_x[i];
+      delete vdofs[i];
+   }
+
```

Comment:
Is this a leak fix that is not yet in master?

Reply:
yes - it is in separate pull request - I guess it will be merged soon in master